resources/js/components/journal/JournalList.vue

```diff
@@ -13,12 +13,14 @@
       <journal-rate-day @hasRated="hasRated" />
 
       <!-- Logs -->
-      <div v-for="journalEntry in journalEntries.data" :key="journalEntry.id" class="cf" :cy-name="'entry-body-' + journalEntry.id">
-        <journal-content-rate v-if="journalEntry.journalable_type == 'App\\Models\\Journal\\Day'" :journal-entry="journalEntry" @deleteJournalEntry="deleteJournalEntry" />
+      <div v-if="journalEntries.data" cy-name="journal-entries-body" :cy-items="journalEntries.data.map(j => j.id)" :cy-object-items="journalEntries.data.map(j => j.object.id)">
```

Comment:
I really don't understand the need of both cy-items and cy-object-item, and I don't understand the map thing.

Reply:
Alright I saw your comment about the map thing below. But why still need to cy data attribute here?

Reply:
I've added a description in the PR to explain this